src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/GrantMandatoryQuestionsController.java

```diff
@@ -27,31 +29,31 @@
 @RequiredArgsConstructor
 public class GrantMandatoryQuestionsController {
 
-    final private GrantMandatoryQuestionService grantMandatoryQuestionService;
+    private final GrantMandatoryQuestionService grantMandatoryQuestionService;
 
-    final private FileService fileService;
+    private final FileService fileService;
 
-    @GetMapping("/scheme/{schemeId}/complete")
-    public ResponseEntity<Boolean> hasCompletedMandatoryQuestions(@PathVariable Integer schemeId) {
-        return ResponseEntity.ok(grantMandatoryQuestionService.hasCompletedMandatoryQuestions(schemeId));
-    }
+    @GetMapping("/scheme/{schemeId}/isCompleted")
```

Comment:
This isn't a RESTful URI name. 

https://restfulapi.net/resource-naming/#:~:text=RESTful%20URI%20should%20refer%20to,Users%20of%20the%20system